VALUES-CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,23 @@
+Our code of conduct are the measures which ensure that we stick to our values
+at every stage of our work and all over at our organisation.
+
+
```

Comment:
Open and proactive communication:
Regarding
- time
- feelings
- solutions to problems
- ideas
- ...